### ISSUE TITLE
8schools generate massive vectors

### ISSUE THREAD

**Issue by Sjnjerak** [stan-dev/rstan#609]
#### Summary:
Please provide a short summary (no more than a sentence or two).
For any basic RStan model (including the 8schools example) I get the error: "Error: cannot allocate vector of size 16761.5 Gb"

#### Description:
Describe the issue as clearly as possible.
I run Windows 10, R 3.5.1, Rstan 2.18.2.
Rstan loads fine but whenever I try to run any model, regardless how simple, I get the error complaining about vector size (e.g. "Error: cannot allocate vector of size 16761.5 Gb"). I understand that I don't have the RAM for these vectors, but I have no idea why they would be this large.

#### Reproducible Steps:
Please report steps to reproduce the issue. If it's not possible to reproduce, please include a description of how you discovered the issue.

If you have a reproducible example, please include it.

As a reproducible example, the 8schools example give me this vector error (any model I can think of does).

#### Current Output:
If applicable, any relevant output from RStan.
When running: "fit <- stan(file = '8schools.stan', data = schools_dat)"
I get:
"Error: cannot allocate vector of size 16761.5 Gb
In addition: Warning message:
In readLines(file, warn = TRUE) :
  incomplete final line found on 'C:\Users\####\OneDrive\Documents\8schools.stan'"

#### Expected Output:
If applicable, the output you expected from RStan.

#### RStan Version:
The version of RStan you are running (e.g., from `packageVersion("rstan")`)
2.18.2

#### R Version:
The version of R you are running (e.g., from `R.version.string`)
3.5.1 (2018-07-02)

#### Operating System:
Your operating system
Windows 10, version 1803

**Comment by bob-carpenter** [comment]
Thanks for reporting.

This is a bug somewhere---the total memory required for 8 schools at run time should be very small.  

When this kind of thing happens, it's often due to mismatched numbers, like 64 bit numbers being interpreted as 32-bit numbers or signed numbers being interpreted as unsigned.

**Comment by Sjnjerak** [comment]
Thanks for the quick reply. I run a 64-bit system and have tried to consistently report 64-bit version of all the software I run.

I think the problem was the cross talk-between R 3.5.1 and Stan, once I updated to R 3.5.2. the issue disappeared. Additional info (in case someone else has the same problem): I ran into this issue because I tried to run R through the Anaconda Eco-system, and Anaconda does not yet support R.3.5.2. for some reason. Manually downloading the most recent version of R and replacing the executable file inside Anaconda fixed everything for me and I can again run RStan inside Jupyter Notebooks.
